### PR TITLE
Set recording comments: Exclude "Add release" page

### PR DESCRIPTION
**Pull request by kellnerd**
Script causes a type error on this page which is unintentionally matched.

### CODE REVIEW

File: set-recording-comments.user.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
 // @match          *://*.mbsandbox.org/release/*
 // @exclude        *musicbrainz.org/release/*/*
 // @exclude        *.mbsandbox.org/release/*/*
```
Comment (murdos) [inline set-recording-comments.user.js:13]: mbsandbox.org doesn't seem to be online anymore: could you remove the two mentions here and on L.11?

File: set-recording-comments.user.js [hunk 2]
```diff
@@ -11,6 +11,7 @@
 // @match          *://*.mbsandbox.org/release/*
 // @exclude        *musicbrainz.org/release/*/*
```
Comment (murdos) [inline set-recording-comments.user.js:12]: Could you update this one so all expressions are similar? (adding prefix `*://`)

File: set-recording-comments.user.js (hunk 1 above)
Comment (kellnerd) [reply set-recording-comments.user.js]: Sure thing, I have now removed them and made the match/exclude patterns consistent. I was not sure whether these sandboxes were internal servers or so and hesitated to remove something that did no harm.

File: set-recording-comments.user.js (hunk 2 above)
Comment (kellnerd) [reply set-recording-comments.user.js]: Sorry, I just remembered that the `@match` patterns do not work for `@exclude` rules. You either have to use the rule as it already is or use `@exclude-match`. Please do not merge yet, I will fix it again!

Comment (kellnerd) [reply set-recording-comments.user.js:11]: I decided to go with `@exclude *musicbrainz.org/release/add*` which will also match possible query parameters while `@exclude-match *://*.musicbrainz.org/release/add` would have been cleaner but unfortunately incompatible with Greasemonkey.

Comment (murdos) [reply set-recording-comments.user.js:11]: Good catch and well done! :+1: 
Is this ready to be merged? It seems so, but I'd rather ask to be sure.

Comment (kellnerd) [reply set-recording-comments.user.js:11]: Yes it is 😇

**Pull request merged**